server/prisma/migrations/20240131062714_update_order_process/migration.sql

```diff
@@ -0,0 +1,212 @@
+  - Added the required column `orderid_fk` to the `donations` table without a default value. This is not possible if the table is not empty.
+  - Made the column `amount` on table `donations` required. This step will fail if there are existing NULL values in that column.
+  - Made the column `frequency` on table `donations` required. This step will fail if there are existing NULL values in that column.
+  - Made the column `eventdescription` on table `events` required. This step will fail if there are existing NULL values in that column.
+  - Made the column `active` on table `events` required. This step will fail if there are existing NULL values in that column.
+  - Made the column `seasonticketeligible` on table `events` required. This step will fail if there are existing NULL values in that column.
+
+*/
+-- DropForeignKey
+ALTER TABLE "donations" DROP CONSTRAINT "donations_donationdate_fkey";
+
+-- DropForeignKey
+ALTER TABLE "donations" DROP CONSTRAINT "donations_donoid_fkey";
+
+-- DropForeignKey
+ALTER TABLE "eventinstances" DROP CONSTRAINT "eventinstances_defaulttickettype_fkey";
+
+-- DropForeignKey
+ALTER TABLE "eventtickets" DROP CONSTRAINT "eventtickets_singleticket_fk_fkey";
+
+-- DropForeignKey
+ALTER TABLE "eventtickets" DROP CONSTRAINT "eventtickets_ticketrestrictionid_fk_fkey";
+
+-- DropForeignKey
+ALTER TABLE "eventtickets" DROP CONSTRAINT "tickets_eventinstanceid_fkey";
+
+-- DropForeignKey
+ALTER TABLE "orderitems" DROP CONSTRAINT "orderitems_orderid_fkey";
+
+-- DropForeignKey
+ALTER TABLE "orders" DROP CONSTRAINT "orders_contactid_fkey";
+
+-- DropForeignKey
+ALTER TABLE "orders" DROP CONSTRAINT "orders_discountid_fkey";
+
+-- DropForeignKey
+ALTER TABLE "orders" DROP CONSTRAINT "orders_orderdate_fkey";
+
+-- DropForeignKey
+ALTER TABLE "seasontickets" DROP CONSTRAINT "seasontickets_eventid_fk_fkey";
+
+-- DropForeignKey
+ALTER TABLE "seasontickets" DROP CONSTRAINT "seasontickets_eventticketid_fk_fkey";
+
+-- DropForeignKey
+ALTER TABLE "seasontickets" DROP CONSTRAINT "seasontickets_orderitemid_fk_fkey";
+
+-- DropForeignKey
+ALTER TABLE "seasontickets" DROP CONSTRAINT "seasontickets_seasontickettypeid_fk_fkey";
+
+-- DropForeignKey
+ALTER TABLE "singletickets" DROP CONSTRAINT "singletickets_orderitemid_fk_fkey";
+
+-- AlterTable
+ALTER TABLE "donations" DROP COLUMN "contactid_fk",
+DROP COLUMN "donationdate",
+DROP COLUMN "donorname",
+DROP COLUMN "isanonymous",
+DROP COLUMN "payment_intent",
+DROP COLUMN "refund_intent",
+ADD COLUMN     "anonymous" BOOLEAN NOT NULL DEFAULT false,
+ADD COLUMN     "orderid_fk" INTEGER NOT NULL,
+ALTER COLUMN "amount" SET NOT NULL,
+ALTER COLUMN "frequency" SET NOT NULL;
+
+-- AlterTable
+ALTER TABLE "eventinstances" DROP COLUMN "defaulttickettype";
+
+-- AlterTable
+ALTER TABLE "events" ALTER COLUMN "eventdescription" SET NOT NULL,
+ALTER COLUMN "eventdescription" SET DEFAULT '',
+ALTER COLUMN "active" SET NOT NULL,
+ALTER COLUMN "active" SET DEFAULT true,
```

Comment:
I imagine that events should be `inactive` by default so they don't appear on the customer site right off the bat.

Reply:
Updated